sympy/geometry/util.py

```diff
@@ -509,6 +511,8 @@ def hypot(x, y):
             if arg.is_Rational:
                 return _sqrt(arg)
             return sqrt(arg)
+    else:
+        from math import hypot
```

Comment:
Tests for this branch?

Reply:
This line is covered by the tests. The `hypot` function was imported at the top of the file and then shadowed in the other branch of the if. I've moved the import here so that you can clearly see what the alternative cases are in the if/else branch.